sortinghat/core/api.py

```diff
@@ -644,6 +714,56 @@ def delete_domain(ctx, domain_name):
     return domain
 
 
+@django.db.transaction.atomic
+def delete_group(ctx, organization, group_name):
```

Comment:
Same comment with the definition of the function as before.

Reply:
Done 👍